src/shared/codelens/csharpCodeLensProvider.ts

```diff
@@ -380,3 +400,60 @@ export function isPublicMethodSymbol(
 export function generateDotNetLambdaHandler(components: DotNetLambdaHandlerComponents): string {
     return `${components.assembly}::${components.namespace}.${components.class}::${components.method}`
 }
+
+export async function installDebugger({
```

Comment:
Does the debugger already exist in the Docker image? If so, can we hijack that instead of downloading ourselves?

Reply:
Nope, it doesn't exist in the image :(